AUTHORS.txt

```diff
@@ -0,0 +1,11 @@
+The following people have contributed to Servus:
+
+-------- git master -------
+
+Stefan Eilemann
```

Comment:
I sorted by my guess on the amount of contribution. If most people agree, I will do it alphabetically.